playwright/utils/expectShell.ts

```diff
@@ -0,0 +1,12 @@
+import { execSync } from 'child_process'
+import { expect } from '@playwright/test'
+
+export function expectShell(cmd: string) {
+  return expect.poll(() => {
```

Comment:
actually does this `poll` make sense in the expect-it-to-say-not-found case? would it say exit code 1 right away? do we actually want to wait for something?

Reply:
Ah, annoyingly good point. I don't see a way to prevent flakiness and false positives here then.

I will make it one-shot. If expect-found becomes flaky after that, then I can add a delay or flush somehow.